ddtrace/settings/_otel_remapper.py

```diff
@@ -0,0 +1,190 @@
+        log.warning("An unrecognized trace exporter '%s' is being used; setting dd_trace_enabled to false.", otel_value)
+        new_value = "False"
+    else:
+        new_value = "True"
+    return new_value
+
+
+def _remap_metrics_exporter(otel_value):
+    """Remaps the otel metrics exporter to ddtrace metrics exporter"""
+    if otel_value != "none":
+        log.warning(
+            "An unrecognized runtime metrics exporter '%s' is being "
+            "used; setting dd_runtime_metrics_enabled to false.",
+            otel_value,
+        )
+    new_value = "False"
+    return new_value
+
+
+def _remap_logs_exporter(otel_value):
+    """Remaps the otel logs exporter to ddtrace logs exporter"""
+    if otel_value != "none":
+        log.warning("Unsupported logs exporter detected.")
+        new_dd_value = ""
+        return new_dd_value
+    return otel_value
+
+
+def _remap_otel_tags(otel_value):
+    """Remaps the otel tags to ddtrace tags"""
+    dd_tags = []
+    otel_tags = otel_value.split(",")
+    otel_user_tag_dict = dict()
+
+    for tag in otel_tags:
+        tag_pair = tag.split("=")
+        otel_user_tag_dict[tag_pair[0]] = tag_pair[1]
+
+    for otel_key, dd_key in OTEL_UNIFIED_TAG_MAPPINGS.items():
+        if otel_key in otel_user_tag_dict.keys():
+            dd_tags.append("{}:{}".format(dd_key, otel_user_tag_dict[otel_key]))
+
+    for key, value in otel_user_tag_dict.items():
+        if len(dd_tags) < 11 and key not in [
+            "deployment.environment",
+            "service.name",
+            "service.version",
+        ]:
+            dd_tags.append("{}:{}".format(key, value))
+
+    if len(otel_user_tag_dict.items()) > 10:
+        ten_tags = dd_tags[:10]
+        log.warning(
+            "To preserve metrics cardinality, only the following first 10 tags have been processed %s",
+            ten_tags,
+        )
+    new_value = ",".join(dd_tags)
+    return new_value
+
+
+def _remap_otel_sdk_config(otel_value):
+    """Remaps the otel sdk config to ddtrace sdk config"""
+    if otel_value == "false":
+        new_value = "True"
+    elif otel_value == "true":
+        new_value = "False"
+    else:
+        log.warning("Unexpected value for OTEL_SDK_DISABLED.")
+        return otel_value
+    return new_value
+
+
+def _remap_default(otel_value):
+    """Remaps the otel default value to ddtrace default value"""
+    return otel_value
+
+
+ENV_VAR_MAPPINGS = {
+    "OTEL_SERVICE_NAME": ("DD_SERVICE", _remap_default),
+    "OTEL_LOG_LEVEL": ("DD_TRACE_DEBUG", _remap_otel_log_level),
+    "OTEL_PROPAGATORS": ("DD_TRACE_PROPAGATION_STYLE", _remap_otel_propagators),
+    "OTEL_TRACES_SAMPLER": ("DD_TRACE_SAMPLE_RATE", _remap_traces_sampler),
+    "OTEL_TRACES_EXPORTER": ("DD_TRACE_ENABLED", _remap_traces_exporter),
+    "OTEL_METRICS_EXPORTER": ("DD_RUNTIME_METRICS_ENABLED", _remap_metrics_exporter),
+    "OTEL_LOGS_EXPORTER": ("none", _remap_logs_exporter),  # This should be ignored
```

Comment:
This mapping feels weird. Would it make sense to map logs exporter to an empty string. We do not want to set "none" accidentally.  Also can we update the doc string in `_remap_logs_exporter` to state that it does not remap to a Datadog config.

```suggestion
    "OTEL_LOGS_EXPORTER": ("", _remap_logs_exporter),  # This should be ignored
```